### ISSUE TITLE
[UX Revamp] Manage owners for package 

### ISSUE THREAD

**Issue by anangaur** [NuGet/NuGetGallery#5226]
The current page to manage owners seems unnecessarily clumsy. This can be made better with few tweaks. 

### Current
![image](https://user-images.githubusercontent.com/14800916/34503089-7c8baab2-efcb-11e7-9ec7-c3db1fc05a9e.png)

### Proposal
![slide2](https://user-images.githubusercontent.com/14800916/34504639-a0a4e428-efd4-11e7-8431-a0a1209ea938.PNG)

![slide3](https://user-images.githubusercontent.com/14800916/34504654-ae697f74-efd4-11e7-8400-271b90392ed2.PNG)

![slide4](https://user-images.githubusercontent.com/14800916/34504662-b6c14de6-efd4-11e7-9459-ac7971477623.PNG)

**Comment by scottbommarito** [comment]
I'm not sure if we should remove the confirmation--even though the user can immediately cancel the request, it still sends an email to the other user. Additionally, if there are any security policies involved (e.g. secure push before it was forced on all users, 2-auth, etc) adding an owner can have a lot of consequences that the user needs to know about.

**Comment by anangaur** [comment]
@scottbommarito  I agree on the security policies related messaging but right we have none. Ones we had have been removed (I guess!).

I do not see any other reason for the confirmation.

**Comment by shishirx34** [comment]
I agree with @scottbommarito , the confirmation is a good idea to have even if the user can be removed immediately, preventing unnecessary email notification.

**Comment by scottbommarito** [comment]
@anangaur We do have policy messaging today, but you likely don't see it because all the accounts you're interacting with have already had their policies added.

(https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Views/Packages/ManagePackageOwners.cshtml#L123)

**Comment by anangaur** [comment]
> @anangaur We do have policy messaging today, but you likely don't see it because all the accounts you're interacting with have already had their policies added.

@scottbommarito thats as good as removed ;-)

@shishirx34 , I disagree that it is useful. At best it prevents an email being sent out. And its only helpful, when I (as NuGet user) add some other user due to spelling mistake inadvertently. There too, what is the chance that I would be able to notice the mistake on confirmation?

IMO, the confirmation should be added where its non trivial or difficult to correct the action. So removing self as owner should prompt as you cannot add yourself. Others actions like adding or even removing other owners should not prompt.
